Rocket.Chat/Controllers/Auth/ConnectServerViewController.swift

```diff
@@ -122,13 +122,31 @@ final class ConnectServerViewController: BaseViewController {
         present(alert, animated: true, completion: nil)
     }
 
+    func normalizeInputURL(_ inputURL: String) -> String? {
+        if inputURL.isEmpty {
+            return nil
+        }
+
+        if let index = inputURL.range(of: "://")?.upperBound {
+            let url = inputURL[index..<inputURL.endIndex]
+            if url.isEmpty { return nil }
+
+            return "https://\(url)"
+        }
+
+        return "https://\(inputURL)"
```

Comment:
I think will be safer here if you transform the URL into URL object and check for the protocol.

Reply:
Will do that, but it's kinda awkward because of this behavior:
https://stackoverflow.com/questions/35960185/nsurlcomponents-builds-scheme-without

URL("open.rocket.chat") means host = nil && path = "open.rocket.chat" 🤣

Reply:
😱